spring-cloud-gcp-docs/src/main/asciidoc/storage.adoc

```diff
@@ -54,7 +54,7 @@ If the resource path refers to an object on Google Cloud Storage (as opposed to
 can be cast as a `GoogleStorageResourceObject` and the `getGoogleStorageObject` method can be called
 to obtain a https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-storage/src/main/java/com/google/cloud/storage/Blob.java[`Blob`].
 This type represents a GCS file, which has associated https://cloud.google.com/storage/docs/gsutil/addlhelp/WorkingWithObjectMetadata[metadata], such as content-type, that can be set.
-
+The `getSignedUrl` method can also be used to obtain signed URLs for GCS objects.
```

Comment:
`getSignedUrl` -> `createSignedUrl`
Also, link to the documentation about signed URLs: https://cloud.google.com/storage/docs/access-control/signed-urls

Reply:
Done